src/ImageSharp/PixelFormats/PixelImplementations/PixelOperations/Rgb24.PixelOperations.cs

```diff
@@ -34,7 +34,7 @@ internal partial class PixelOperations : PixelOperations<Rgb24>
                 int count = redChannel.Length;
                 GuardPackFromRgbPlanes(greenChannel, blueChannel, destination, count);
 
-                SimdUtils.PackFromRgbPlanes(configuration, redChannel, greenChannel, blueChannel, destination);
+                SimdUtils.PackFromRgbPlanes(redChannel, greenChannel, blueChannel, destination);
```

Comment:
It looks like we can also remove the `configuration` argument in the next major release?